diff --git a/node_modules/json-schema-to-typescript/dist/src/parser.js b/node_modules/json-schema-to-typescript/dist/src/parser.js
index c9c6779..89de019 100644
--- a/node_modules/json-schema-to-typescript/dist/src/parser.js
+++ b/node_modules/json-schema-to-typescript/dist/src/parser.js
@@ -318,7 +318,7 @@ function parseSchema(schema, options, processed, usedNames, parentSchemaName) {
             const ast = parse(value, options, key, processed, usedNames);
             const comment = `This interface was referenced by \`${parentSchemaName}\`'s JSON-Schema definition
 via the \`patternProperty\` "${key.replace('*/', '*\\/')}".`;
-            ast.comment = ast.comment ? `${ast.comment}\n\n${comment}` : comment;
+            ast.comment = ast.comment ? `${ast.comment}\n\n${comment}` : '';
             return {
                 ast,
                 isPatternProperty: !singlePatternProperty,
diff --git a/node_modules/json-schema-to-typescript/src/parser.ts b/node_modules/json-schema-to-typescript/src/parser.ts
index 92acdeb..14009ee 100644
--- a/node_modules/json-schema-to-typescript/src/parser.ts
+++ b/node_modules/json-schema-to-typescript/src/parser.ts
@@ -400,7 +400,7 @@ function parseSchema(
         const ast = parse(value, options, key, processed, usedNames)
         const comment = `This interface was referenced by \`${parentSchemaName}\`'s JSON-Schema definition
 via the \`patternProperty\` "${key.replace('*/', '*\\/')}".`
-        ast.comment = ast.comment ? `${ast.comment}\n\n${comment}` : comment
+        ast.comment = ast.comment ? `${ast.comment}\n\n${comment}` : ''
         return {
           ast,
           isPatternProperty: !singlePatternProperty,
